pdfjinja.py

```diff
@@ -129,10 +129,28 @@ def check(self, data):
         return bool(data)
 
     def paste(self, data):
-        rect = self.context["rect"]
+        rect = self.context["rect"] 
         x, y = rect[0], rect[1]
         w, h = rect[2] - x, rect[3] - y
-        pdf = self.Attachment(data, dimensions=(x, y, w, h)).pdf()
+        # Changed to accomodate StringIO
+        img = Image.open(imageData)
```

Comment:
We should move this functionality to the Attachment class instead and pass parameters to it from here.